docs/guides/start.md

```diff
@@ -80,6 +80,9 @@ promiseService.send('RESOLVE');
 // => 'resolved'
 ```
 
-You can also copy/paste the `Machine(...)` code and [visualize it on XState Viz](https://xstate.js.org/viz):
+You can also visualize the code by changing `createMachine` to `Machine` and copy-pasting it into [the XState Viz](https://xstate.js.org/viz):
```

Comment:
I'm not sure what's the current status of https://github.com/statecharts/xstate-viz but maybe we could redeploy it with this patch to avoid adding notes like this and making this easier for the users?
```diff
diff --git a/src/StateChart.tsx b/src/StateChart.tsx
index f3b9a9e..ad69112 100644
--- a/src/StateChart.tsx
+++ b/src/StateChart.tsx
@@ -141,6 +141,7 @@ export function toMachine(machine: StateNode<any> | string): StateNode<any> {
   try {
     createMachine = new Function(
       'Machine',
+      'createMachine',
       'interpret',
       'assign',
       'send',
@@ -165,6 +166,7 @@ export function toMachine(machine: StateNode<any> | string): StateNode<any> {
 
   try {
     createMachine(
+      machineProxy,
       machineProxy,
       interpret,
       assign,
```

Reply:
Would definitely be less cognitive load for the user! Should I take that block out for now and have this PR as pending till the xstate viz gets updated?

Reply:
Let's wait for @davidkpiano to see if releasing the Visualizer is even an option.

Reply:
@davidkpiano anything I can do to help you on this one? is there going to be a major docs restructuring that just makes it moot maybe?